src/uploadx/src/uploadx.service.ts

```diff
@@ -55,17 +63,14 @@ export class UploadxService {
   async handleFile(file: File) {
```

Comment:
It looks like all Uploadservice methods are not asynchronous now

Reply:
yes you are right, I fix that to